draft-demarco-oauth-status-assertions.md

```diff
@@ -576,7 +576,7 @@ The Status Assertion MUST contain the parameters defined below.
 | **exp** | UNIX Timestamp with the expiration time of the JWT. It MUST be greater than the value set for `iat`. | {{RFC9126}}, {{RFC7519}}, {{RFC7515}} |
 | **credential_hash** | Hash value of the Digital Credential the Status Assertion is bound to. | this specification |
 | **credential_hash_alg** | The Algorithm used of hashing the Digital Credential to which the Status Assertion is bound. The value SHOULD be set to `sha-256`. | this specification |
-| **credential_status_validity**| Boolean value indicating the absolute validity of the Credential linked to the Status Assertion. This parameter is REQUIRED, and the Verifier MUST verify its presence and value to assess the Credential's validity. | this specification |
+| **credential_status_validity**| Numerical value indicating the validity of the Credential linked to the Status Assertion, describing its state, mode, condition or stage. The value MUST be from the IANA registry (TBD). Status validity parameter is REQUIRED, and the Verifier MUST verify its presence and value to assess the Credential's validity. | this specification |
```

Comment:
```suggestion
| **credential_status_validity**| Numerical value indicating the validity of the Credential linked to the Status Assertion, describing its state, mode, condition or stage. The value MUST be from the IANA registry (TBD).Values from 0 to 2 are defined by this specification with the following meaning: 0-VALID, 1-INVALID, 2-SUSPENDED. Credential Issuers MAY extend with other numeric values based on use cases of interest. Status validity parameter is REQUIRED, and the Verifier MUST verify its presence and value to assess the Credential's validity. | this specification |
```

Reply:
I think that we agreed that we don't want to hardcode the values in this spec, as we want to align values with related parameter in Status List specification. We assume that they will register the values in the IANA registry so we want to reference to that instead. I think thought that we can use this values in the examples.

Reply:
Even if we will have the values registered in IANA, why shouldn't we have in this specification? IMHO having them explicitly here could be useful for implementers that are already implementing this. Am I missing something?

Reply:
I created this issue, https://github.com/peppelinux/draft-demarco-oauth-status-assertions/issues/90

and am resolving this conversation.